pyproject.toml

```diff
@@ -15,7 +15,8 @@ exclude = '''
 target-version = "py39"
 line-length = 79
 exclude = [".git", "venv"]
-ignore = ["E501", "D10", "D400", "D401", "D415", "D205", "B904", "SIM118"]
+[tool.ruff.lint]
+ignore = ["E501", "D10", "D400", "D401", "D415", "D205", "B904", "SIM118", "D203", "D213"]
 select = [
```

Comment:
Should we use [`extend-select`](https://docs.astral.sh/ruff/settings/#lint_extend-select) instead of [`select`](https://docs.astral.sh/ruff/settings/#lint_select) so that the default rules are automatically enabled?